shuup/core/models/_products.py

```diff
@@ -694,15 +703,15 @@ def make_package(self, package_def):
                     _("Variation parents can not belong into a package"),
                     code="abnormal"
                 )
-            if child_product.is_package_parent():
+            if child_product.is_container():
                 raise ImpossibleProductModeException(_("Packages can't be nested"), code="multilevel")
             if quantity <= 0:
                 raise ImpossibleProductModeException(_("Quantity %s is invalid") % quantity, code="quantity")
             ProductPackageLink.objects.create(parent=self, child=child_product, quantity=quantity)
         self.verify_mode()
 
     def get_package_child_to_quantity_map(self):
-        if self.is_package_parent():
+        if self.is_container():
```

Comment:
Your commits are not properly ordered, since I remember seeing usages of `is_container` before it's being added.